main.tf

```diff
@@ -84,15 +86,15 @@ resource "aws_instance" "default" {
   #bridgecrew:skip=BC_AWS_GENERAL_31: Skipping `Ensure Instance Metadata Service Version 1 is not enabled` check until BridgeCrew supports conditional evaluation. See https://github.com/bridgecrewio/checkov/issues/793
   count                       = local.instance_count
   ami                         = data.aws_ami.info.id
-  availability_zone           = local.availability_zone
+  availability_zone           = var.availability_zone
```

Comment:
@aknysh is this is the latest best practice to ensure ebs_optimized is `true`?

Reply:
I think this is fair; we should change the default to EBS optimized as the best practice around performance. It can always be disabled (e.g. for cost purposes).

Reply:
@tirumerla  let's switch to EBS optimized